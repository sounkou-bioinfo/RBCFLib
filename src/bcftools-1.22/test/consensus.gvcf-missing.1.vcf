##fileformat=VCFv4.2
##contig=<ID=1,length=100>
##contig=<ID=2,length=100>
##contig=<ID=3,length=100>
##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	1
1	3	.	A	<*>	.	.	END=5	GT	./.
2	3	.	A	.	.	.	END=5	GT	./.
3	3	.	A	.	.	.	.	GT	./.
3	4	.	A	.	.	.	.	GT	./.
3	5	.	A	.	.	.	.	GT	./.
