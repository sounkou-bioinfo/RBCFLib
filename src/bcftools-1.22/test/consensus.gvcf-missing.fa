>1
AAAAAA
>2
AAAAAA
>3
AAAAAA
