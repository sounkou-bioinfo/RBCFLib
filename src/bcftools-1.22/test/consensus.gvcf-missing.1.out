>1
AANNNA
>2
AANNNA
>3
AANNNA
